src/yamlToAnalytics.ts

```diff
@@ -130,11 +130,12 @@ export const buildFileContents = async (inDir: string) => {
         .concat(tss)
         .concat(Object.keys(jsonzWithSubstituteRefs)
             .map((key) => jsonzWithSubstituteRefs[key].title)
-            .map((title) =>
-`export const make${capNoSpace(title)} = (userId?: string, properties?: ${capNoSpace(title)}) => ({
+            .map((title) =>  // tslint:disable-next-line
+`export const make${capNoSpace(title)} = ({ userId, anonymousId, properties }: { userId?: string, anonymousId?: string, properties: ${capNoSpace(title)} }) => ({
     userId,
```

Comment:
`userId` to either remain as is or change as per what we agree in the other PR :innocent: